test/__snapshots__/test.js.snap

```diff
@@ -1,5 +1,320 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`CEReactions.webidl 1`] = `
+"\\"use strict\\";
+
+const conversions = require(\\"webidl-conversions\\");
+const utils = require(\\"./utils.js\\");
+
+const impl = utils.implSymbol;
+
+class CEReactions {
+  constructor() {
+    throw new TypeError(\\"Illegal constructor\\");
+  }
+
+  method() {
+    // CEReaction pre steps
```

Comment:
This seems to be a bit inconsistent: the brand check is outside the code passed to the user-defined hook for accessors. The [spec](https://html.spec.whatwg.org/multipage/custom-elements.html#cereactions) says

> Operations, attributes, setters, or deleters annotated with the `[CEReactions]` extended attribute must run the following steps **in place of the ones specified in their description**

which seems to me that the brand/type checking and other Web IDL conversion code should not be included as part of the hook.

E.g., for deleters:

```diff
       deleteProperty(target, P) {
-        // CEReaction pre steps
 
         if (typeof P === \\"symbol\\") {
           return Reflect.deleteProperty(target, P);
         }
 
         if (target[impl][utils.supportsPropertyName](P) && !(P in target)) {
+        // CEReaction pre steps
           target[impl][utils.namedDelete](P);
+        // CEReaction post steps
           return true;
         }
 
         return Reflect.deleteProperty(target, P);
 
-        // CEReaction post steps
       },
```

Reply:
Good catch. This is not as clear as it could be in the spec; see https://github.com/whatwg/html/issues/3879. But @TimothyGu is right on the intent of the spec.